### ISSUE TITLE
PresentationCore.dll is not optimized so can't be used for benchmarking

### ISSUE THREAD

**Issue by dotMorten** [dotnet/wpf#1894]
* .NET Core Version: 3.0.100-preview9-014004
* Windows version: 1903, 1809
* Does the bug reproduce also in WPF for .NET Framework 4.8?: No
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? No

 **Problem description:**

When using BenchmarkDotNet to run some benchmarks and the WPF Libraries are used, BenchmarkDotNet calls out with the following error:
```
// Validating benchmarks:
Assembly bdn which defines benchmarks references non-optimized PresentationCore
        If you own this dependency, please, build it in RELEASE.
        If you don't, you can disable this policy by using 'config.With(ConfigOptions.DisableOptimizationsValidator)'.
```

  **Actual behavior:** 
 Non-optimized warning
 **Expected behavior:**
 No warning
 **Minimal repro:**

`Program.cs`: 
```cs
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Running;

namespace bdn
{
    class Program
    {
        static void Main(string[] args)
        {
             var summary = BenchmarkRunner.Run<Benchmarks>();
        }
    }
	
    public class Benchmarks
    {
        [Benchmark]
        public object SimpleBenchmark() => new System.Windows.Media.Color();
    }
}
```

`BDN.csproj`:
```xml
<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp3.0</TargetFramework>
    <UseWpf>true</UseWpf>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="BenchmarkDotNet">
      <Version>0.11.5</Version>
    </PackageReference>
  </ItemGroup>
</Project>

```

**Comment by rladuca** [comment]
This was fixed for GA via #1549.  The latest daily builds of [Release/3.0](https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries) should work fine.

![image](https://user-images.githubusercontent.com/20006987/64926281-9bd2a500-d7b0-11e9-8d5c-a070df43317f.png)

**Comment by fabiant3** [comment]
Closing, @dotMorten thank you for reporting.
